File: Case_1/src/typescriptServer/util/Timer.ts
export default class Timer {

    private begin : number;
    private end : number;

    constructor() {
        this.begin = new Date().getTime();
        this.end = new Date().getTime();   
    }

    public start(){
        this.begin = new Date().getTime();
    }

    public stop(){
        this.end = new Date().getTime(); 
    }

    public getTime() : Number {
        return this.end - this.begin;
    }

}

File: Case_1/src/dotnetServer/Utils/Timer.cs
using System;
class Timer
{
    private DateTime begin;
    private DateTime end;

    public Timer(){
        this.begin = DateTime.Now;
        this.end = DateTime.Now;
    }

    public void start(){
        this.begin = DateTime.Now;
    }

    public void stop(){
        this.end = DateTime.Now;
    }

    public double getTime(){
        return ((TimeSpan)(this.end - this.begin)).TotalMilliseconds;
    }

}